### PR TITLE
[fsinternetradio] Add Ruark R5 to tested devices & mode list

### PR DESCRIPTION
**Pull request by grssnbchr**
Signed-off-by: Timo Grossenbacher <timo@timogrossenbacher.ch>

# [fsinternetradio] Add Ruark R5 to tested devices & mode list

# Description

This merely updates the README with the Ruark R5 model since I own one of these and tested it successfully with that binding.

### CODE REVIEW

File: bundles/org.openhab.binding.fsinternetradio/README.md [hunk 1]
```diff
@@ -68,20 +69,21 @@ All devices support some of the following channels:
...
+| Silvercrest SIRD 14 C2   | Internet Radio | TIDAL                   | Deezer       | Qobuz        | Spotify   | -        | Music Player | DAB Radio    | FM Radio  | AUX in    |-       | - | - | - |
+| Auna KR200 Kitchen Radio | Internet Radio | Spotify                 | -            | Music Player | DAB Radio | FM Radio | AUX in       | -            | -          | -         |-       | - | - | - |
+| Auna Connect CD          | Internet Radio | Spotify                 | -            | Music Player | DAB Radio | FM Radio | CD           | Bluetooth    | AUX in    | -         | -      | - | - | - |
+| Teufel 3sixty            | Internet Radio | Spotify                 | -            | USB/Network  | DAB Radio | FM Radio | Bluetooth    | AUX in       | -          | -         | -      | - | - | - |
+| Ruark R5                 | Internet Radio | TIDAL                   | Deezer       | Amazon Music | Spotify   | Local Music | Music Player | DAB Radio | FM Radio   | Bluetooth | AUX in  | Phono | Optical | CD
```
Comment (lolodomo) [inline bundles/org.openhab.binding.fsinternetradio/README.md:86]: Please add the missing `|` at the end of this added line.

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by grssnbchr** [comment]
Thank you for your feedback. 
Sorry, I forgot the DCO with this latest commit. Should I make another commit on top?

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
> Sorry, I forgot the DCO with this latest commit. Should I make another commit on top?

I merged even if DCO check was wrong as your PR is only a small enhancement of the documentation.
Thank you.

**Pull request merged**